helm/prometheus-meta-operator/templates/pod-monitor.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: monitoring.coreos.com/v1
+kind: PodMonitor
+metadata:
+  name: {{ include "resource.default.name" . }}
+  namespace: {{ include "resource.default.namespace" . }}
+  labels:
+    name: {{ include "resource.default.name" . }}
+spec:
+  namespaceSelector:
+    matchNames:
+      - {{ include "resource.default.namespace" . }}
+  selector:
+    matchLabels:
+      {{- include "labels.selector" . | nindent 6 }}
+  podMetricsEndpoints:
+    - targetPort: 8000
```

Comment:
Do you have to use the port number there, or could you use its name `http` ?

Reply:
We cannot as the port is not exposed in the deployment but that's doable if we expose the port and name it